changes/3be2f29a5e433cb1cf122d69571a3d23.yaml

```diff
@@ -0,0 +1,7 @@
+---
+desc: Updated the ``indent`` keyword argument of ``$lib.json.save()`` to be a boolean
```

Comment:
Nit, but i'm not really sure this is a "feature". I wonder if we should add a "notes" section to the release docs?

Reply:
I updated the changelog tool with a "note" type and changed this entry to be a "note". Should be good now.